src/components/Calendars/ClientScheduleAppointment.tsx

```diff
@@ -0,0 +1,432 @@
+                                                    >
+                                                        {}
+                                                    </div>
+                                                </div>
+                                            );
+                                        })}
+                                        {no_of_days.map((day, index) => {
+                                            return (
+                                                <div className="px-0.5 mb-2" key={index}>
+                                                    <div
+                                                        key={index}
+                                                        onClick={() => {
+                                                            if (!hasNoAvailability(day)) selectDateValue(day);
+                                                        }}
+                                                        className={
+                                                            hasNoAvailability(day)
+                                                                ? 'text-center text-base rounded-lg leading-loose w-9 h-9 font-skylight text-decoration-line: line-through'
+                                                                : 'cursor-pointer text-center text-base rounded-lg leading-loose w-9 h-9 font-skylight'
+                                                        }
+                                                        style={{
+                                                            backgroundColor: hasNoAvailability(day)
+                                                                ? 'rgb(236, 237, 239)'
+                                                                : isSelectedDate(day)
+                                                                ? 'rgb(243, 252, 243)'
+                                                                : 'white',
+
+                                                            color: isToday(day)
+                                                                ? 'rgb(88, 148, 66)'
+                                                                : hasNoAvailability(day)
+                                                                ? 'rgb(149, 155, 167)'
+                                                                : 'black',
+
+                                                            border: hasNoAvailability(day)
+                                                                ? '1px solid rgb(236, 237, 239)'
+                                                                : isSelectedDate(day)
+                                                                ? '1px solid rgb(88, 148, 66)'
+                                                                : '1px solid rgb(149, 155, 167)',
+                                                        }}
```

Comment:
again, dont use style and dont invent colors that aren't already defined by tailwind